src/routes/get-all.ts

```diff
@@ -0,0 +1,13 @@
+import { Router } from "express";
+const router = Router();
+import { posts } from "../posts";
+
+router.get("/api/posts", (req, res) => {
+  if (posts.length === 0) {
```

Comment:
The spider returning posts should return an empty array if there are no posts to return.

Reply:
done